components/Typography/src/heading.scss

```diff
@@ -19,4 +19,18 @@
 .utrecht-heading-6 {
   /* TODO: Move to container element, make spacing optional */
   --utrecht-space-around: 1;
+
+  font-family: var(--denhaag-typography-sans-serif-alternate);
+}
+
+@media (max-width: 768px) {
+  .utrecht-heading-1 {
+    --utrecht-heading-1-font-size: 2.25rem;
+  }
+  .utrecht-heading-2 {
+    --utrecht-heading-2-font-size: 1.75rem;
+  }
+  .utrecht-heading-3 {
+    --utrecht-heading-3-font-size: 1.4375rem;
+  }
```

Comment:
I don't think you need to specify the component, you could manage the responsive theme in a separate file:

```css
@media (max-width: 768px) {
   .denhaag-theme {
     --utrecht-heading-1-font-size: 2.25rem;
     --utrecht-heading-2-font-size: 1.75rem;
     --utrecht-heading-3-font-size: 1.4375rem;
   }
}
```

By separating the theme from the component, it remains usable as white-label component for other teams.

Taking it one step further, I think it would be good to specify the entire mobile scale from Figma in `typography.tokens.json` so you can use:

```css
@media (max-width: 768px) {
   .denhaag-theme {
     --utrecht-heading-1-font-size: var(--denhaag-typography-mobile-scale-3xl-font-size);
     --utrecht-heading-2-font-size: var(--denhaag-typography-mobile-scale-2xl-font-size);
     --utrecht-heading-3-font-size: var(--denhaag-typography-mobile-scale-xl-font-size);
   }
}
```

Reply:
@Robbert I applied your feadback into the newest commit.

Reply:
Nice! Eventually (in another PR) we will need to find a way to combine all these responsive `denhaag-theme` customizations in one file, so they can be shipped along with the design tokens. @Yolijn any ideas?

Reply:
🤔 Good idea @Robbert I don't have a thought out idea at the moment, but let's brainstorm about it this week!